documentation/slate/source/includes/interface.md

```diff
@@ -126,3 +126,29 @@ That means :
 * where you can find the details of your object ( **"id": "edc46f3a-ad3d-11e4-a5e1-005056a44da2"** ).
 
 
+
+Objects order
+------------
+
+Unless specified otherwise objects are not sorted and stability of object's order is not guaranteed.
```

Comment:
would prefer
"Unless specified, objects are not sorted and stability of object's order is not guaranteed."
no?